scripts/build.sh

```diff
@@ -76,4 +76,5 @@ cp ./build/distributions/*.zip $OUTPUT/plugins
 ./gradlew publishAllPublicationsToStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
 ./gradlew publishPluginZipPublicationToZipStagingRepository -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
 mkdir -p $OUTPUT/maven/org/opensearch
-cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch
\ No newline at end of file
+cp -r ./build/local-staging-repo/org/opensearch/. $OUTPUT/maven/org/opensearch
+cp -r ./build/local-staging-repo/org/opensearch/plugin/opensearch-job-scheduler ~/.m2/repository/org/opensearch/plugin/
```

Comment:
@owaiskazi19 please add a comment linking the skills repo issue.

Reply:
^^

Reply:
Done